tappsbt/mock.go

```diff
@@ -88,6 +89,7 @@ func RandPacket(t testing.TB) *VPacket {
 		}},
 		Outputs: []*VOutput{{
 			Amount:                             123,
+			AssetVersion:                       asset.Version(rand.Intn(2)),
```

Comment:
Need to use `test.RandIntn(2)` here to get stable/deterministic test vectors. Also, nit: line too long.